app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_elements = command.split()
+    _command = command_elements[0]
+    file_to_copy = command_elements[1]
+    file_to_write = command_elements[2]
```

Comment:
```suggestion
    _command, file_to_copy, file_to_write = command_elements
```

Reply:
Extract elements from the list returned by command.split() and assign them to variables.

Reply:
You should check the length of elements before unpacking to avoid unexpected errors